CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This CHANGELOG follows the format located [here](https://github.com/sensu-plugin
 
 ### Changed
 - sensuctl cli args for asset updates now uses `--namespace`
+- use stable package channels (@webframp)
```

Comment:
this is a breaking change since it uses completely different packages and not all platforms supported in the alpha/beta are currently supported.

Reply:
Moved this to breaking changes section, thanks